src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs

```diff
@@ -299,7 +299,16 @@ public DatabaseData(CDataContainer context, string databaseName)
 
                 ResourceManager manager = new ResourceManager("Microsoft.SqlTools.ServiceLayer.Localization.SR", typeof(DatabasePrototype).GetAssembly());
 
-                this.owner = db.Owner;
+                try
+                {
+                    this.owner = db.Owner;
+                }
+                catch (Exception)
+                {
+                    // TODO: fix the exception in SMO
```

Comment:
yes 

---
In reply to: [122092979](https://github.com/Microsoft/sqltoolsservice/pull/380#discussion_r122092979) [](ancestors = 122092979)